integration_tests/dbt_project.yml

```diff
@@ -29,6 +29,10 @@ vars:
     jira_user_identifier: "user"
     jira_version_identifier: "version"
 
+models:
+  jira_source:
+    +schema: "{{ 'sqlw_tests' if target.name == 'databricks-sql' else 'jira' }}"
```

Comment:
Reminder to update the schema name to be unique to Jira for the SQL Warehouse tests